c7n/resources/appelb.py

```diff
@@ -80,6 +80,7 @@ class resource_type(TypeInfo):
         arn = "LoadBalancerArn"
         # The suffix varies by type of loadbalancer (app vs net)
         arn_type = 'loadbalancer/app'
+        supports_trailevents = True
```

Comment:
this shouldn't be needed, supports_trailevents is default true except for child resources where it is false.

Reply:
Removed this change